inc/namespace.php

```diff
@@ -124,10 +128,27 @@ function override_two_factor_universally_forced( bool $is_forced ) : bool {
  * @return array|null
  */
 function override_two_factor_forced_user_roles( $roles ) {
+	if ( maybe_skip_two_factor_forcing_for_local_env() ) {
+		return [];
```

Comment:
As we're still loading it I could configure the different roles in the admin intentionally but this would override that. I think this should just return `$roles` as-is unless the plugin has default values for including admins.

Reply:
Note that the `two-factor` plugin saves settings in DB, so returning `$roles` here might enforce 2fa on local still, against what the filter is meant to do.

Reply:
TBH developers can always configure this separately for local, either via config or options, so I'll go with your suggestion to follow the cloud module practice. It's better to be consistent, for the sake of future incidents as well.